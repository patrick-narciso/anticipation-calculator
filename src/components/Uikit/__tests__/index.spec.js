import tokens from '../Tokens';

test('Ensures no design system changes', () => {
  expect(tokens).toMatchSnapshot();
});
