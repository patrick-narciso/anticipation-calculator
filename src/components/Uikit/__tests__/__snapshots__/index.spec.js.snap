// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ensures no design system changes 1`] = `
DesignSystem {
  "ds": Object {
    "breakpoints": Object {
      "sm": "768px",
      "xs": "415px",
      "xxs": "320px",
    },
    "colors": Object {
      "brand": Object {
        "black": "#656565",
        "white": "#FFFFFF",
      },
      "colorPalette": Object {
        "blue": Object {
          "dark": "#3D75BB",
          "light": "#5D9CEC",
        },
        "grey": Object {
          "dark": "#F2F2F2",
          "light": "#CECECE",
        },
      },
    },
    "type": Object {
      "regularFontFamily": "Source Sans Pro, sans-serif",
      "sizes": Object {
        "base": "18px",
        "l": "24px",
        "s": "16px",
        "xl": "48px",
        "xs": "14px",
        "xxs": "11px",
      },
    },
  },
  "opts": Object {
    "fontSizeUnit": "rem",
  },
}
`;
